From a168b8509a0d952a7ae9d15f5ee641569a766e77 Mon Sep 17 00:00:00 2001
From: Anatol Derksen <derksen@fs-net.de>
Date: Thu, 18 Aug 2022 19:28:45 +0200
Subject: [PATCH] Add domain permission for UART2 on M7

Add domain permission for UART2 on M7
Add UART2 read/write PDAP permission for the Cortex-M7.
- like commit  72eaab22df7bdb832c374fb94bd15b437bd3397e
---
 plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
index b0b08fa..5dba419 100644
--- a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
@@ -60,7 +60,7 @@ static const struct imx_rdc_cfg rdc[] = {
 	RDC_MDAn(RDC_MDA_M7, DID1),
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
+	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W | D1R | D1W),
 	RDC_PDAPn(RDC_PDAP_WDOG1, D0R | D0W),
 
 	/* memory region */
-- 
2.21.3

